api/tests/common/test_app.rs

```diff
@@ -370,6 +370,9 @@ impl TestApp {
 }
 
 pub async fn spawn_test_app() -> TestApp {
+    // We set the environment to dev.
+    Environment::Dev.set();
```

Comment:
Fixed this since I forgot to run the entire pipeline in the previous PR and missed the failures.